test/e2e/app/math.go

```diff
@@ -0,0 +1,10 @@
+package app
+
+func Sum(batches []int) uint64 {
```

Comment:
Sum doesn't need to be exported.
```suggestion
func sum(batches []int) uint64 {
```